From akpm@osdl.org Tue Mar 28 14:47:55 2006
Message-Id: <200603282247.k2SMleVc029572@shell0.pdx.osdl.net>
From: Vivek Goyal <vgoyal@in.ibm.com>
Subject: PCI: 64 bit resources more sound changes
To: vgoyal@in.ibm.com, greg@kroah.com, mm-commits@vger.kernel.org
Date: Tue, 28 Mar 2006 14:47:34 -0800

From: Vivek Goyal <vgoyal@in.ibm.com>

- Changes required to fix compilation warnings for sound/* dir for 64bit
  resources.  These changes came up due to cross-compilation on powerpc with
  CONFIG_PPC32=y

Signed-off-by: Vivek Goyal <vgoyal@in.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 sound/ppc/pmac.c |   14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/sound/ppc/pmac.c
+++ gregkh-2.6/sound/ppc/pmac.c
@@ -1198,9 +1198,10 @@ int __init snd_pmac_new(struct snd_card 
 					       chip->rsrc[i].start + 1,
 					       rnames[i]) == NULL) {
 				printk(KERN_ERR "snd: can't request rsrc "
-				       " %d (%s: 0x%08lx:%08lx)\n",
-				       i, rnames[i], chip->rsrc[i].start,
-				       chip->rsrc[i].end);
+				       " %d (%s: 0x%016llx:%016llx)\n",
+				       i, rnames[i],
+				       (unsigned long long)chip->rsrc[i].start,
+				       (unsigned long long)chip->rsrc[i].end);
 				err = -ENODEV;
 				goto __error;
 			}
@@ -1229,9 +1230,10 @@ int __init snd_pmac_new(struct snd_card 
 					       chip->rsrc[i].start + 1,
 					       rnames[i]) == NULL) {
 				printk(KERN_ERR "snd: can't request rsrc "
-				       " %d (%s: 0x%08lx:%08lx)\n",
-				       i, rnames[i], chip->rsrc[i].start,
-				       chip->rsrc[i].end);
+				       " %d (%s: 0x%016llx:%016llx)\n",
+				       i, rnames[i],
+				       (unsigned long long)chip->rsrc[i].start,
+				       (unsigned long long)chip->rsrc[i].end);
 				err = -ENODEV;
 				goto __error;
 			}
